cluster/nats_rpc_server.go

```diff
@@ -52,6 +52,7 @@ type NatsRPCServer struct {
 	bindingsChan           chan *nats.Msg // bindingsChan receives notify from other servers on every user bind to session
 	unhandledReqCh         chan *protos.Request
 	userPushCh             chan *protos.Push
+	userKickCh             chan *nats.Msg
```

Comment:
Why did we change the standard here? you should probably use `protos.KickMsg` instead of `nats.Msg` and `processKick` would be more like `processPushes`. I am aware that the way it is now will work, but I'd rather keep the standards we set before. 

Since this was already merged, could you open another PR changing this, please? @gabrielerzinger